sunpy/coordinates/frameattributes.py

```diff
@@ -53,14 +53,14 @@ def convert_input(self, value):
         if value is None:
             return None, False
 
-        elif value == 'now':
-            return Time(datetime.datetime.now()), True
-
         elif isinstance(value, Time):
             out = value
             converted = False
 
         elif isinstance(value, str):
+            if value == 'now':
+                return Time(datetime.datetime.now()), True
```

Comment:
Could we put this shortcut in `parse_time()`?

Reply:
I think it is?

Reply:
https://github.com/sunpy/sunpy/blob/master/sunpy/time/time.py#L317